src/core/observables/CylindricalLBFluxDensityProfileAtParticlePositions.hpp

```diff
@@ -0,0 +1,109 @@
+#ifndef OBSERVABLES_CYLINDRICALLBFLUXDENSITYPROFILEATPARTICLEPOSITIONS_HPP
+#define OBSERVABLES_CYLINDRICALLBFLUXDENSITYPROFILEATPARTICLEPOSITIONS_HPP
+
+#include "CylindricalProfileObservable.hpp"
+#include "utils.hpp"
+#include "utils/Histogram.hpp"
+#include "lbgpu.hpp"
+
+namespace Observables {
+class CylindricalLBFluxDensityProfileAtParticlePositions : public CylindricalProfileObservable {
```

Comment:
Please move the implementation to a .cpp file. Then you don't have to drag in the other headers here, which are not actually part of the interface. That also decreases the build time.